### PR TITLE
US and other location import fix 

### PR DESCRIPTION
**Pull request by webmods-croatia**
https://github.com/navjottomer/Osclass/issues/85

### CODE REVIEW

**Comment by webmods-croatia** [comment]
@navjottomer wm-patch6 branch is made from your release-3.9.0 branch, and this PR targets release-3.9.0. Should I change both to develop?

**Comment by navjottomer** [comment]
> @navjottomer wm-patch6 branch is made from your release-3.9.0 branch, and this PR targets release-3.9.0. Should I change both to develop?

Nope, release branch will be merged to develop and develop into master, when we think release branch is production ready.

File: oc-includes/osclass/helpers/hUtils.php [hunk 1]
```diff
@@ -322,6 +322,7 @@ function osc_get_locations_json() {
  * @return string
  */
 function osc_get_locations_sql($location) {
+    $location = str_replace(' ', '%20', $location);
```
Comment (navjottomer) [inline oc-includes/osclass/helpers/hUtils.php]: Isn't better to use urlencode( $location)

Comment (webmods-croatia) [reply oc-includes/osclass/helpers/hUtils.php]: I've had problems with URL encode in this situations so I didn't use it.

**Comment by webmods-croatia** [comment]
@navjottomer Check my answer to change requested above.

File: oc-includes/osclass/helpers/hUtils.php (hunk 1 above)
Comment (navjottomer) [reply oc-includes/osclass/helpers/hUtils.php]: Use rawurlencode it will stick to the standard . Let me know if you have any specific issue with it.

Comment (webmods-croatia) [reply oc-includes/osclass/helpers/hUtils.php]: @navjottomer That did it. I updated the branch.

**Pull request merged**